{"name": "test_handle_api_call_exceptions_async_request_error", "status": "passed", "attachments": [{"name": "log", "source": "9caf166c-da12-444e-8ce1-f8e130d6e2e0-attachment.txt", "type": "text/plain"}], "start": 1753152649975, "stop": 1753152649975, "uuid": "6d4af438-aab8-4550-8947-271d71cebf45", "historyId": "c65b132b15b6f233aaddbe04cf9ecbd7", "testCaseId": "c65b132b15b6f233aaddbe04cf9ecbd7", "fullName": "tests.unit.utils.test_decorator#test_handle_api_call_exceptions_async_request_error", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "parentSuite", "value": "tests.unit.utils"}, {"name": "suite", "value": "test_decorator"}, {"name": "host", "value": "pkrvmq0rgcvqdmg"}, {"name": "thread", "value": "2923-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.unit.utils.test_decorator"}]}